.github/workflows/ci.yml

```diff
@@ -70,23 +70,27 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
+          - "3.13"
     name: Python ${{ matrix.python-version }} - Full
     steps:
       - uses: actions/checkout@v4
 
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+          allow-prereleases: true
 
-      - name: Requirements check
-        run: python -m pip list
+      - uses: astral-sh/setup-uv@v3
+
+      - name: Install nox
+        run: uv tool install nox
```

Comment:
Should we `uvx`/install `nox` using `uv` consistently across this file, or is there a reason to switch between these 2 in different jobs?

Reply:
I think one just used it on more lines. We can normalize to either one if that's better.

Reply:
Ah, I see, that makes sense. Leaving it as it is looks okay. Thanks!